src/components/ThePostHeader.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <div class="post">
+    <div class="title">
+      {{ post.title }}
+      <button class="title__postbutton">
+        이전글
+      </button>
+      <button class="title__postbutton">
+        다음글
+      </button>
+      <button class="title__listbutton">
+        목록
+      </button>
+    </div>
+    <div id="metadata">
+      <img :src="userPictureUrl" class="post-author-profile-picture"/>
+      <div class="post-header">
+        <div class="post-header__author">
+          {{ postAuthor }}
+        </div>
+
+        <div class="post-header__info">
+          <span class="post-header__time">
+            {{ postCreatedAt }}
+          </span>
+
+          <PostStatus class="post-header__status" :post="post" />
```

Comment:
The "vote" action should be handled. I think we need to modify some lines of PostStatus (handle votable prop and vote event on LikeButton, if the user want)